Automattic/wp-calypso
# Marketplace: Adds Categories Responsive Menu

#### Screencast
![Screen Capture on 2022-05-16 at 20-28-33](https://user-images.githubusercontent.com/1035546/168709024-1b1c9f89-dc94-4431-a67e-4732758a7c55.gif)

#### Changes proposed in this Pull Request

* Adds the `ResponsiveToggleGroup` to `@automattic/components`.
* Adds the responsive categories menu into the marketplace.

#### Testing instructions

* Navigate into the marketplace
* Test multiple categories, check everything is working fine.

Related to #62891
